styles/style.css

```diff
@@ -65,5 +65,6 @@ body {
 }
 .copy{
   font-size: 10px;
-  margin: 0px 0px 16px;
+  margin: 0px 0px 16px 0px;
```

Comment:
you dont have to change it, but just letting you know that if ur adding for only one, you can just use margin-bottom: 16px, if you want